### PR TITLE
Adding Commit Messages Section

### PR DESCRIPTION
**Pull request by harrythomas1998**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,3 +130,44 @@ Overly complex structures and unnecessary code can be avoided by effective refac
...
+[Link 3](https://www.linkedin.com/pulse/how-write-very-bad-commit-messages-ran-bar-zik/)
+
+
+**Bad Commit Messages:**
+
+![Bad Commit Messages](badcommit.png)
+
+**Good Commit Messages:**
+
+[Good Commit Messages](goodcommit.png)
```
Comment (mikeygillen) [inline README.md:176]: This image is not showing for me it is just a link is this the same for everyone?

File: README.md [hunk 2]
```diff
@@ -130,3 +130,44 @@ Overly complex structures and unnecessary code can be avoided by effective refac
...
+*	Atomize(break up into small units)
+*    End the subject line with a period
+
+If you want to read more on good commit messages use these links:
+
+[Link 1](https://medium.com/@andrewhowdencom/anatomy-of-a-good-commit-message-acd9c4490437)
+
+[Link 2](https://chris.beams.io/posts/git-commit/)
+
+[Link 3](https://www.linkedin.com/pulse/how-write-very-bad-commit-messages-ran-bar-zik/)
```
Comment (mikeygillen) [inline README.md]: Give headings to these links don't just label them Link 1, 2 etc.

**Comment by mikeygillen** [comment]
Very good work overall, I like the layout and the information is informative if you could just change the link titles I think we could merge.

File: README.md (hunk 1 above)
Comment (harrythomas1998) [reply README.md:176]: The image hasn't been uploaded yet, it will be there once merged

File: README.md (hunk 2 above)
Comment (harrythomas1998) [reply README.md]: Will Do

**Comment by chriskavanagh97** [comment]
yeah looks good

**Pull request merged**